examples/miscellaneous/miscellaneous_ex12/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+#set -x
+
+source $LIBMESH_DIR/examples/run_common.sh
+
+example_name=miscellaneous_ex12
+
+options="-pc_type lu -pc_factor_mat_solver_package mumps"
```

Comment:
Do you know if this example works without a direct solver and/or without PETSc?  We'll have to programmatically restrict this example to PETSc-enabled builds otherwise.